ios/Podfile

```
@@ -51,14 +51,23 @@ target app_target do
 end
 
 # ---- Hermes Fix 2: Remove "Replace Hermes" script phases everywhere ----
-FORBIDDEN_HERMES_MARKERS = ['Replace Hermes', 'Replace Hermes for the right configuration'].freeze
+# Include both the explicit "Replace Hermes" scripts and any stray phases
+# named with bracketed [Hermes] tags.
+FORBIDDEN_HERMES_MARKERS = [
+  'replace hermes',
+  'replace hermes for the right configuration',
+  '[hermes]'
```

RuboCop (1.76.1)

[convention] Put a comma after the last item of a multiline array.

(Style/TrailingCommaInArrayLiteral)